test/testutils.jl

```diff
@@ -28,8 +28,9 @@ end
 # testing the implementation of a discrete univariate distribution
 #
 function test_distr(distr::DiscreteUnivariateDistribution, n::Int;
-                    testquan::Bool=true)
-
+                    testquan::Bool=true, rng::AbstractRNG=MersenneTwister(),
+                    test_scalar_rand::Bool=false)
+    println("    testing $(distr)")
```

Comment:
I think tests are already quite verbose? If needed, they can also be grouped/wrapped in a `@testset`.
```suggestion
```